webapp-django/crashstats/topcrashers/jinja2/topcrashers/topcrashers.html

```diff
@@ -198,11 +198,11 @@ <h2>
                     <td>{{ topcrashers_stats_item.num_crashes_per_platform.mac_count }}</td>
                     <td>{{ topcrashers_stats_item.num_crashes_per_platform.lin_count }}</td>
                   {% elif os_name == 'Windows' %}
-                    <td>{{ topcrashers_stats_item.win_count }}</td>
+                    <td>{{ topcrashers_stats_item.num_crashes_per_platform.win_count }}</td>
```

Comment:
`topcrashers_stats_item.win_count` doesn't exist. Everything is in the `num_crashes_per_platform` property of the `SignatureStats` instance.